sparkle/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@dust-tt/sparkle",
-  "version": "0.2.640",
+  "version": "0.2.641",
```

Comment:
you don't need to manually bump the sparkle version, you can bump + publish from github action 🙏https://www.notion.so/dust-tt/Runbook-Sparkle-development-10a28599d94180c1bde9cfbbfcba6ed3?source=copy_link#27928599d9418057b523effa80fb6ef5

Reply:
oh nice! Will do